app/Http/Controllers/Actions/Approvals/ApproveApproval.php

```
@@ -18,30 +23,36 @@
      */
     public function __invoke($modelType, $model, Request $request)
     {
-        $result = $model->updateApproval([
-            'status' => RequestApprovalStatus::APPROVED,
-            'date_approved' => Carbon::now(),
-        ]);
-
+        $cacheKey = "approve" . $modelType . $model->id. '-'. Auth::user()->id;
+        if (Cache::has($cacheKey)) {
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Cache' in method '__invoke'. (undefined)

(StaticAccess)